__tests__/push/bundler.test.ts

```diff
@@ -46,18 +46,27 @@ async function validateZip(content) {
 
   const files: Array<string> = [];
 
-  const entries = createReadStream(pathToZip).pipe(
-    unzipper.Parse({ forceStream: true })
-  );
-
-  let targetFileContent = null;
-  for await (const entry of entries) {
-    files.push(entry.path);
+  console.log(partialPath);
+
+  let targetFileContent = '';
+  await new Promise(r => {
+    createReadStream(pathToZip)
+      .pipe(unzipper.Parse())
+      .on('entry', function (entry) {
+        const fileName = entry.path;
+        files.push(fileName);
+        if (fileName === partialPath) {
+          entry.on('data', d => (targetFileContent += d));
+        } else {
+          entry.autodrain();
+        }
+      })
+      .on('close', () => {
+        r(null);
+      });
```

Comment:
We can also just resolve here:

```suggestion
      .on('close', r)
```